inst/rmarkdown/templates/isba/resources/template.tex

```diff
@@ -0,0 +1,163 @@
+% based on https://vtex-soft.github.io/texsupport.isba-ba/
+% \documentclass[ba,preprint]{imsart}% use this for supplement article
+\documentclass[ba]{imsart}
```

Comment:
I see in the comment 
````
% \documentclass[ba,preprint]{imsart}% use this for supplement article
````

So I understood that `preprint` is an option that can be passed. 
Without the `classoption` variable it can't. 

If I misunderstood, please feel free to ignore. 

And in general, not all my remarks require change if you don't think that worth it. You know better this format and you are the user.